ts/components/wallet/card/ActionIcon.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Component that renders an icon from the
+ * icon font and wraps it with a touchable
+ * highlight (thus allowing onPress actions)
+ */
+import * as React from "react";
+import { View } from "react-native";
+import { StyleSheet, TouchableHighlight } from "react-native";
+import Icon from "../../../theme/font-icons/io-icon-font/index";
+import variables from "../../../theme/variables";
+
+type Props = Readonly<{
+  name: string;
+  onPress?: () => void;
+  size?: number;
+  touchable?: boolean;
+}>;
+
+const styles = StyleSheet.create({
+  iconWrapper: {
+    paddingLeft: 10
+  }
+});
+
+export default class ActionIcon extends React.Component<Props> {
+  public static defaultProps = {
+    size: variables.iconSizeBase,
+    touchable: true
+  };
+
+  public render() {
+    const { onPress } = this.props;
+    const icon = (
+      <Icon
+        name={this.props.name}
+        color={variables.brandPrimary}
+        size={this.props.size}
+      />
+    );
+    return this.props.touchable ? (
+      <TouchableHighlight
```

Comment:
I am quite sure IconProps extends TextProps so onPress is already available.

Reply:
I followed the chain of type definitions (Icon from react-native-vector-icon -> IconProps -> TextProps -> onPress) and I would agree with you; but then when trying to pass an `onPress` property, I get a "Property 'onPress' does not exist on type ...".

I'm probably missing something here, any clue as to what that is?

Also, I'll be switching to the component defined in #222 when available, so it might make sense to address this problem there. 

For the moment, since I don't actually need the component to be touchable (ever since implementing the popup menu), I can get rid of this